src/latexify/frontend.py

```diff
@@ -55,6 +58,8 @@ def get_latex(
         tree = transformers.IdentifierReplacer(identifiers).visit(tree)
     if reduce_assignments:
         tree = transformers.AssignmentReducer().visit(tree)
+    if prefixes is not None:
+        tree = transformers.PrefixTrimmer(prefixes).visit(tree)
```

Comment:
We have a pre-defined list of prefixes: 

https://github.com/google/latexify_py/blob/3b1b05bc5d6d9811799327dc268001e94bf68a28/src/latexify/constants.py#L5

This list is used in `FunctionCodegen` to trim the prefix, but we can employ this list for the input of `PrefixTrimmer` as well, and remove the ad-hoc implementation in `FunctionCodegen`.

```suggestion
    # assume that both prefixes and PREFIXES are sets.
    # merge and de-duplicate the prefix list.
    merged_prefixes = constants.PREFIXES | prefixes
    tree = transformers.PrefixTrimmer(merged_prefixes).visit(tree)
```